source/dem/particle_wall_dmt_force.cc

```diff
@@ -179,29 +101,109 @@ ParticleWallDMTForce<dim>::calculate_particle_wall_contact_force(
             ((particle_properties[DEM::PropertiesIndex::dp]) * 0.5) -
             (projected_vector.norm());
 
-          if (normal_overlap > 0)
+          // Minimal delta_star
+          if (normal_overlap > dmt_cut_off_distance)
             {
-              contact_information.normal_overlap = normal_overlap;
-
-              this->update_contact_information(contact_information,
-                                               particle_location_3d,
-                                               particle_properties,
-                                               dt);
-
-              // This tuple (forces and torques) contains four elements which
-              // are: 1, normal force, 2, tangential force, 3, tangential torque
-              // and 4, rolling resistance torque, respectively
+              // i is the particle, j is the wall.
+              // we need to put a minus sign in front of the
+              // normal_vector to respect the convention (i ->
+              // j)
+              const unsigned int particle_type =
+                particle_properties[DEM::PropertiesIndex::type];
+              const double effective_radius =
+                0.5 * particle_properties[DEM::PropertiesIndex::dp];
+              const double effective_surface_energy =
+                this->effective_surface_energy[particle_type];
+              const double effective_hamaker_constant =
+                this->effective_hamaker_constant[particle_type];
+
+              const double delta_0 =
+                -effective_hamaker_constant / (75.39822368615503 * // 24. * M_PI
```

Comment:
Personal opinion, but I would use something like
const double pi_24 = 75.39822368615503; // 24 * M_PI
and all those kind of constants at the begining of the function. 
Idk, it feels cleaner

Reply:
agreed